### PR TITLE
Custom pod labels

### LINKED ISSUE #563: [Feature Request ] Add support for custom pod labels
We have several global network policies based on labels, currently the only way that I can make the awx accessible is edit the awx deployment after being created by the operator and set the correct label for allows access. Would it be possible to set the awx pod label directly on the operator ?

This request is similar to the https://github.com/ansible/awx-operator/issues/269 , but that one is about labeling the service, not the awx pod.

### PR DESCRIPTION
**Pull request by philipsd6**
This should resolve #563. If/when this passes muster, I'll squash so it can be merged.

### CODE REVIEW

File: roles/installer/defaults/main.yml
Comment (shanemcd) [inline roles/installer/defaults/main.yml]: Did you mean `scrape`?

Comment (philipsd6) [reply roles/installer/defaults/main.yml]: Yes, sure did -- fixed, and PR updated.

**Comment by shanemcd** [comment]
Hi, sorry for the slow review. Looks like this has conflicts now. Once you fix, we'll get this reviewed and merged.

**Comment by philipsd6** [comment]
> Hi, sorry for the slow review. Looks like this has conflicts now. Once you fix, we'll get this reviewed and merged.

No problem @shanemcd -- I've gone ahead and rebased on devel without conflicts as requested, and squashed down to one commit.

EDIT: whoops, I forgot to pull upstream before rebasing, and my fork wasn't up-to-date, so I'll get that fixed quickly...

**Review by tchellomello — APPROVED** [review]
Very nice @philipsd6!! Thanks for your contribution!!

**Comment by jsalatiel** [comment]
Any chance we can get this merged ?

**Comment by timansky** [comment]
I am having the same problem. #1160 must resolve issue. waiting for merge

**Review by erz4 — APPROVED** [review]

**Comment by fosterseth** [comment]
additional labels was implemented in this PR https://github.com/ansible/awx-operator/pull/1160

does that PR address the use case you had in mind @philipsd6

**Comment by philipsd6** [comment]
@fosterseth I believe it does, as PR #1160 enables setting `additional_labels` which as I understand it would then apply (from the example in my README) update `prometheus.io/scrape-awx` to the pods, which was our use-case for this PR. However, the `additional_labels` solution will also apply it to the service as well, but I can't see why that would be a problem, so I would consider our use-case met by PR #1160.

**Pull request closed without merging**
